src/file.rs

```diff
@@ -302,6 +302,39 @@ impl error::Error for Error {
     }
 }
 
+/// A raw deserializable log4rs configuration for xml.
+#[cfg(feature = "xml_format")]
+#[derive(Deserialize)]
+#[serde(deny_unknown_fields)]
+pub struct RawConfigXml {
```

Comment:
Why do we need different config types for XML?

Reply:
Unfortunately serde does not allow to use a colon character in xml tag. However it is useful to use this in loggers. So instead of declaring loggers like this:
```xml
<loggers>
    <foo::bar::bazz ... /> <-- serde does not like this tag
```
you have to declare them like this:
```xml
    <loggers>
        <logger name="foo::bar::baz" ... />
```

Reply:
Ah ok that makes sense.